src/ucp/tag/rndv.c

```diff
@@ -559,7 +563,7 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_get_completion, (self, status),
                                                send.state.uct_comp);
 
     if (rndv_req->send.state.dt.offset == rndv_req->send.length) {
-        ucp_rndv_complete_rma_get_zcopy(rndv_req);
+        ucp_rndv_complete_rma_get_zcopy(rndv_req, status);
```

Comment:
should not we call `ucp_rndv_complete_rma_get_zcopy` even if `dt.offset != send.length`, but status is not ok?

Reply:
in this case it should be completed from pending queue , by ucp_ep_err_pending_purge

Reply:
@brminich FYI

Reply:
ok